README.md

```diff
@@ -100,12 +106,27 @@ Before you get started, the following needs to be installed:
   bundle exec rake jobs:work
   ```
 
-1. In a new console, open the project root folder and start the server. The simplest way is to use the included Webrick server:
+
+1. Starting the development server in a new console (open the project root folder)
+Some components are created with React and they need to be build with foreman
+
+  1. Hot Reloading of Rails Assets
 
   ```bash
-  bundle exec rails server
+  foreman start -f Procfile.hot
   ```
 
+  1. Static Loading of Rails Assets
+  ```bash
+  foreman start -f Procfile.static
+  ```
+
+  1. Creating Assets for Tests
+  ```bash
+  foreman start -f Procfile.spec
+  ```
+
```

Comment:
I agree, this could be improved a bit:

* Use imperative mood. `s/Starting the development.../Start the development...`
* Would it make sense to pick one of these and move others under "Advanced setup" paragraph and "Testing" paragraph.